### PR TITLE
Fix redirection and pageload logging

### PR DESCRIPTION
**Pull request by idegtiarov**
Currently button to switch on correct problem is removed and student is redirected to correct problem automatically.

Logging of loading page is done only when student is allowed to see problem content.

### CODE REVIEW

File: static/js/server_interaction.js [hunk 1]
```diff
@@ -143,7 +151,7 @@
 
                         if (data['next'] == null) {
                             //The server just hasn't set the next problem yet. Wait and retry (for 5s)
-                            if (retry < 5) {
+                            if (retry < 2) {
```
Comment (z4y4ts) [inline static/js/server_interaction.js]: Are you sure 2 seconds is enough? If yes, update comment above.

File: static/js/textbox.html [hunk 2]
```diff
@@ -0,0 +1,113 @@
+<html>
+<head>
+
+    <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.7.1/jquery.min.js" type="text/javascript"></script>
+    <script src="https://edx-stanford-dev.raccoongang.com/course-content/2017_N/static/js/server_interaction.js" type="text/javascript"></script>
+    <!-- <script src="http://52.210.53.36:8080/static/server_interaction.js" type="text/javascript"></script> -->
```
Comment (z4y4ts) [inline static/js/textbox.html]: Delete commented code

File: static/js/textbox.html [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+                //We're in the wrong place, redirect to correct page
+                window.top.location.href = e.detail.redirect
+            }
+            resizeIframe();
+        }
+
+        function display_problem_error_callback(e)
+        {
+            var text_div = document.getElementById('text');
+            text_div.innerHTML = "Error connecting to RL server.</br> Probably you are not enrolled in Adapt application, if it is your case visit Adapt enrolling page for that section, usually 'Introduction' or connect to the support team if automatic enrollment is not provided.</br> Details: " + e.detail.err + "</br>" + e.detail.textStatus;
```
Comment (z4y4ts) [inline static/js/textbox.html]: It looks like nobody knows what `RL` stands for. So I guess we should change it to `EdxAdapt` here and everywhere to at least not confuse ourselves.
Not in this PR though.

File: static/js/server_interaction.js (hunk 1 above)
Comment (idegtiarov) [inline static/js/server_interaction.js]: Done

File: static/js/textbox.html (hunk 2 above)
Comment (idegtiarov) [inline static/js/textbox.html]: done

**Pull request merged**